test-suite/testheaders.sh

```diff
@@ -36,9 +35,9 @@ for f in $@; do
 int main( int argc, char* argv[] ) { return 0; }
 EOF
         if ${cc} -c -o $t.o $t.cc ; then
-            echo "Ok."
+            echo "Testing ${f} ... Ok."
```

Comment:
I know that the current PR code preserves the official output, but I think we should adjust that output to better match the new/post-PR reality where the results are reported _after_ the header has been tested. For example:

```suggestion
            echo "PASS: ${f}"
```

The above variant uses output syntax from other/regular tests:

```
PASS: tests/testMem
PASS: tests/testYesNoNone
PASS: tests/testDns
PASS: tests/testEnumIterator
```

I do not insist on this or any specific variant.